cli/internal/context/context.go

```diff
@@ -277,7 +277,7 @@ func (c *Context) populateTopologicGraphForPackageJSON(pkg *fs.PackageJSON, root
 	pkg.TransitiveDeps = []string{}
 	seen := mapset.NewSet()
 	var lockfileWg sync.WaitGroup
-	c.resolveDepGraph(&lockfileWg, pkg.UnresolvedExternalDeps, externalDepSet, seen, pkg)
+	c.resolveDepGraph(&lockfileWg, pkg.Name, pkg.UnresolvedExternalDeps, externalDepSet, seen, pkg)
```

Comment:
We should be passing down `pkg.Dir.ToUnixPath().ToString()` since both pnpm and Yarn identify workspace projects by path and not by name.

Reply:
Other thing we need to do is to special case on when `pkg` is the root package and pass `.`

Reply:
Switched to `pkg.Dir` and `turbopath.AnchoredUnixPath`